api/utilconversion/conversion_fuzz.go

```diff
@@ -16,6 +16,7 @@ package utilconversion
 import (
 	"math/rand"
 
+	//nolint:depguard
```

Comment:
I am curious why we are hitting this.  Is this now included in the allowed imports? Or this is because of deprecation etc?

Reply:
```
utilconversion/conversion_fuzz.go:19:2: import 'github.com/onsi/gomega' is not allowed from list 'main': do not import testing packages in non-test sources (depguard)
	"github.com/onsi/gomega"
```
I didn't rename this file to `conversion_test.go` as it's not for testing `conversion.go`.